runtime/recipe/view.js

```diff
@@ -101,13 +104,15 @@ class View {
 
   isResolved() {
     assert(Object.isFrozen(this));
-    return (this._id || this._create) && this._type;
+    if (this._fate === "copy" && this.id === null)
```

Comment:
this is a bit hard to read and understand.
maybe a
switch (this._fate) {
  case "copy": return !this.id===null && this._type;
  ...
}

this is more verbose, but i think, easier to follow (and update, if/when needed)

ps: also maybe you could use the Copy const you defined at the top of the file here too.